### PR TITLE
Changelog for release 0.14.0

### PR DESCRIPTION
**Pull request by faderskd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +28,7 @@ Fix improper mapping for tracking mode in subsription edit page.
 
 Added options to trace only discarded messages for subscription and error for topic.
 
-### Enhancments
+### Enhancements
```
Comment (cristaloleg) [inline CHANGELOG.md]: The dark power of copypasta

Comment (faderskd) [reply CHANGELOG.md]: ;)

File: docs/docs/configuration/kafka-and-zookeeper.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ zookeeper.authorization.enabled  | n/a                            | enable Zooke
 zookeeper.authorization.scheme   | storage.authorization.scheme   | authorization scheme                                                       | digest
 zookeeper.authorization.user     | storage.authorization.user     | username                                                                   | user
 zookeeper.authorization.password | storage.authorization.password | password                                                                   | password
+zookeeper.max.inflight.requests  | n/a                            | maximum number of unacknowledged requests before blocking                  | 10
```
Comment (faderskd) [inline docs/docs/configuration/kafka-and-zookeeper.md]: https://cwiki.apache.org/confluence/display/KAFKA/KIP-214%3A+Add+zookeeper.max.in.flight.requests+config+to+the+broker

File: hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java [hunk 3]
```diff
@@ -16,7 +16,7 @@
     ZOOKEEPER_MAX_RETRIES("zookeeper.max.retries", 29),
     ZOOKEEPER_CONNECTION_TIMEOUT("zookeeper.connection.timeout", 10000),
     ZOOKEEPER_SESSION_TIMEOUT("zookeeper.session.timeout", 10000),
-    ZOOKEEPER_MAX_INFLIGHT_REQUESTS("zookeeper.max.inflight.requests", 1),
+    ZOOKEEPER_MAX_INFLIGHT_REQUESTS("zookeeper.max.inflight.requests", 10),
```
Comment (faderskd) [inline hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java]: https://cwiki.apache.org/confluence/display/KAFKA/KIP-214%3A+Add+zookeeper.max.in.flight.requests+config+to+the+broker

**Pull request merged**